### PR TITLE
feat: mllp outbound

### PR DESCRIPTION
**Pull request by erick-sunarto**
finalize mllp_outbound feature

testing on my coder:
image

### CODE REVIEW

File: mllp_http/http2mllp.py [hunk 1]
```diff
@@ -1,17 +1,17 @@
+import functools
 import http.server
 import logging
 import socket
 import threading
 import time
-from .mllp import write_mllp
-from .net import read_socket_bytes
+from .mllp import read_mllp,write_mllp_socket
+from .net import read_real_socket_bytes
```
Comment (pauldraper) [inline mllp_http/http2mllp.py:9]: Will you format with black?

(Or run with `make format` but that will also require some npm deps.)

File: mllp_http/http2mllp.py [hunk 2]
```diff
@@ -68,57 +69,42 @@ def send(self, data):
...
+    def close(self):
         self.closed = True
         self.socket.close()
 
-    def send(data):
-        write_mllp(self.socket, data)
-        self.socket.flush()
+    def send(self,data):
+        write_mllp_socket(self.socket, data)
+        # self.socket.flush()
```
Comment (pauldraper) [inline mllp_http/http2mllp.py:95]: What is the purpose of this change?

File: mllp_http/http2mllp.py [hunk 3]
```diff
@@ -68,57 +69,42 @@ def send(self, data):
...
-
-
-class MllpConnection:
-    def __init__(self, socket):
-        self.address = address
         self.cancel = None
         self.closed = False
-        self.responses = read_mllp(read_socket_bytes(self.socket))
         self.socket = socket
+        self.responses = read_mllp(read_real_socket_bytes(self.socket))
```
Comment (pauldraper) [inline mllp_http/http2mllp.py:85]: What is the purpose of this change?

**Review by pauldraper — CHANGES_REQUESTED** [review]

File: mllp_http/http2mllp.py (hunk 1 above)
Comment (erick-sunarto) [reply mllp_http/http2mllp.py:9]: new commit formatted with black. sorry my vscode somehow doesn't auto format python.

File: mllp_http/http2mllp.py (hunk 2 above)
Comment (erick-sunarto) [reply mllp_http/http2mllp.py:99]: currently, http2mllp uses socket vs stream that mllp2http uses. so it require slightly different read/write function to send/receive

File: mllp_http/http2mllp.py (hunk 3 above)
Comment (erick-sunarto) [reply mllp_http/http2mllp.py:89]: currently, http2mllp uses socket vs stream that mllp2http uses. so it require slightly different read/write function to send/receive

**Review by nathanmoon — APPROVED** [review]
Looks like some good cleanup and fixes. Thanks.
